examples/quick-start-artillery/docker-compose.yml

```diff
@@ -0,0 +1,147 @@
+version: "3.5"
+name: pokeshop
+
+services:
+  tracetest-agent:
+    environment:
+      TRACETEST_DEV: ${TRACETEST_DEV}
+      TRACETEST_API_KEY: ${TRACETEST_AGENT_API_KEY}
+      TRACETEST_SERVER_URL: ${TRACETEST_SERVER_URL}
```

Comment:
This is not specified in the .env file. Should we keep or remove it? It might confuse users.